### PR TITLE
New Special - jackInTheBox

### PR DESCRIPTION
**Pull request by rpemberton**
Demo: http://codepen.io/RPem/pen/JRPaEq

My first ever pull request so just let me know if there's anything I've missed.

### CODE REVIEW

**Comment by WarenGonzaga** [comment]
Wow this is cool!

**Comment by eltonmesquita** [comment]
@rpemberton, to me it looks a lot like the Tada animation. Any thoughts on this? I might be missing something.

File: source/specials/jackInTheBox.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  to {
+  	opacity: 1;
+  	transform: scale(1);
+  }
+}
+
+.jackInTheBox {
+  -webkit-animation-name: jackInTheBox;
```
Comment (daneden) [inline source/specials/jackInTheBox.css]: We should only be writing unprefixed properties in the source CSS

Comment (rpemberton) [inline source/specials/jackInTheBox.css]: Thanks, that's now removed

**Comment by WarenGonzaga** [comment]
Hey @daneden  I think we can add this one in the animate.css

**Comment by rpemberton** [comment]
I've just added a commit as I wasn't sure if I'd done it correctly the first time and wanted to be sure.

**Comment by eltonmesquita** [comment]
Thanks @daneden and @rpemberton !
I'm gonna merge this.

**Comment by rpemberton** [comment]
Thanks @eltonmesquita I've just created a PR for the website too #694

**Pull request merged**